bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -0,0 +1,97 @@
+# TapoControl Binding
+
+This binding adds support to control Tapo (Copyright © TP-Link Corporation Limited) Smart Home Devices from your local openHAB system.
+(Note: This is not an official Tapo/TP-link Binding!)
+
+## Supported Things
+
+The following Tapo-Devices are supported
+
+### P100 SmartPlug (WiFi)
```

Comment:
You could mention the Thing Type UIDs, needed for textual configuration.

Reply:
textual configuration is shown under in "Full Example"